pkg/utils/strings.go

```diff
@@ -25,3 +30,33 @@ func StringsContain(items []string, source string) bool {
 	}
 	return false
 }
+
+// CompareSlices will compare two string slice, with the three return values: [both in A and B], [only in A], [only in B]
+func CompareSlices(a []string, b []string) ([]string, []string, []string) {
```

Comment:
CompareSlices is a bit ambiguous. Maybe ThreeWaySliceCompare or SliceIntersect?

Reply:
ThreeWaySliceCompare wins